contrib/windows/juliarc.jl

```diff
@@ -1,6 +1,11 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 let user_data_dir
+    try
+        run(`stty`)
+    catch
+        ENV["PATH"] = joinpath(JULIA_HOME,"..","mintty","usr","bin")*";"*ENV["PATH"]
+    end
```

Comment:
I think this path hack may screw things up for those who build their own julia and use it in place.